src/main/java/se/kth/sda/skeleton/exception/ResourceNotFoundException.java

```diff
@@ -0,0 +1,8 @@
+package se.kth.sda.skeleton.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.NOT_FOUND)
+public class ResourceNotFoundException extends RuntimeException {
+}
```

Comment:
I notice this is inside package called "exception" , I think when Giang & I were working, I noticed a package called "exeption" . Was this a replacement?

Reply:
No, we created also one. Can you just move yours from the "posts" package?

Reply:
Ideally, we should created a small PR before, where we add this exception, PostRepository and annotations in the Post model and then split from there, but what to do. We'll swim out of it somehow...